spec/features/merchants/dashboard_spec.rb

```diff
@@ -7,10 +7,10 @@
     @merchant_3 = Merchant.create!(name: 'Tom Haverford')
     @merchant_4 = Merchant.create!(name: 'April Ludgate')
 
-    @item_1 = @merchant_1.items.create!(name: "Necklace", description: "A thing around your neck", unit_price: 100)
-    @item_2 = @merchant_1.items.create!(name: "Bracelet", description: "A thing around your neck", unit_price: 100)
-    @item_3 = @merchant_1.items.create!(name: "Earrings", description: "A thing around your neck", unit_price: 100)
-    @item_4 = @merchant_1.items.create!(name: "Gauges", description: "A thing around your neck", unit_price: 100)
+    @item_1 = @merchant_1.items.create!(name: "Necklace", description: "A thing around your neck", unit_price: 100, status: 0)
+    @item_2 = @merchant_1.items.create!(name: "Bracelet", description: "A thing around your neck", unit_price: 100, status: 0)
+    @item_3 = @merchant_1.items.create!(name: "Earrings", description: "A thing around your neck", unit_price: 100, status: 0)
+    @item_4 = @merchant_1.items.create!(name: "Gauges", description: "A thing around your neck", unit_price: 100, status: 1)
```

Comment:
Was this change necessary for a test in the dashboard_spec?

Reply:
Nope, that was an accident and I changed it haha.